### PR TITLE
[#155037639] Add spruce to cf-cli for manifest templating

### PR DESCRIPTION
**Pull request by keymon**
What?
----

I want to generate the manifest of the product page, pulling  secrets from .yml files in S3 buckets as we still don't have any sensible proper solution for this.

spruce[1] is a nice tool to kickly generate manifests, pull secrets from other variables, etc. It is handy to have it in the cf-cli container to simplify our pipelines, so that we can build a manifest and add the required config (env vars, secrets, routes, etc) seamlessly.

We also want to add bash, as it has more capabilities than the simple `sh`, we can write simpler scripts. The additional consume sized is minimal.

[1] https://github.com/geofffranks/spruce

How to review?
--------------

Code review

Who?
---

Anyone but me

### CODE REVIEW

File: cf-cli/Dockerfile [hunk 1]
```diff
@@ -28,3 +29,10 @@ RUN curl -L -o "${CF_AUTOPILOT_TEMPFILE}" \
   && chmod +x "${CF_AUTOPILOT_TEMPFILE}" \
   && cf install-plugin -f "${CF_AUTOPILOT_TEMPFILE}" \
   && rm "${CF_AUTOPILOT_TEMPFILE}"
+
+RUN apk add --update wget ca-certificates \
```
Comment (bandesz) [inline cf-cli/Dockerfile]: ca-certificates is already installed and probably you should not remove it at the end

Comment (bandesz) [reply cf-cli/Dockerfile]: also curl is available so, use that instead of wget

**Comment by keymon** [comment]
comments fixed

**Comment by keymon** [comment]
> see my comments

Fixed

**Pull request merged**